presto-orc/src/main/java/com/facebook/presto/orc/TupleDomainFilter.java

```diff
@@ -494,25 +496,30 @@ public String toString()
         }
     }
 
-    class BigintValues
+    class BigintValuesUsingHashTable
             extends AbstractTupleDomainFilter
     {
         private static final long EMPTY_MARKER = 0xdeadbeefbadefeedL;
         // from Murmur hash
         private static final long M = 0xc6a4a7935bd1e995L;
 
+        private final long min;
+        private final long max;
```

Comment:
It looks like the `values` array can be removed. We can address it in a separate PR.